src/shared/environment/Environment.php

```diff
@@ -38,6 +38,14 @@ public function getPathToCommand(string $command): Filename {
 
     abstract public function hasHomeDirectory(): bool;
 
+    public function getPhiveHomeVariable(): string {
+        return $this->getVariable('PHIVE_HOME');
+    }
+
+    public function hasPhiveHomeVariable(): bool {
+        return $this->hasVariable('PHIVE_HOME');
+    }
+
```

Comment:
I decided to place these method next to `hasHomeDirectory()` as they are somehow related.